### DISCUSSION TITLE
How to trigger I/O manually using the api?

### DISCUSSION THREAD

**Discussion by koatheking** [AxisCommunications/discussions#241] [VAPIX] [answered]
Is there an example of how to set up an i/O port for a A1601 controller such that we can open/close said relay using the api?

**Comment by onie5** [comment]
You can use VAPIX to trigger IO in the A1601 however you will need to use firmware version below 10. Upgrading to version 10 and above will disable VAPIX.

**Reply by koatheking** [reply]
We are using version 1.84.3
Is there a guide somewhere on how to use the api to trigger an IO

**Comment by onie5** [comment, marked as answer]
Please check this out:

https://www.axis.com/vapix-library/subjects/t10175981/section/t10074527/display


Ronaldo
Axis Communications
________________________________

**Reply by koatheking** [reply]
Thank you. I was looking only in the "Physical Access Control" section because the A1601 is a door controller. These commands worked.
